examples/chat/module/rust/src/lib.rs

```diff
@@ -57,22 +58,22 @@ impl OakNode for Node {
     }
 }
 
-fn unknown_id_err(which: &str) -> grpc::Result<()> {
+fn unknown_id_err(which: &str) -> grpc::Result<Empty> {
```

Comment:
I tried that but it messes up the trait bounds &ndash; they're expecting `T: protobuf::Message`.  It seemed nicer to have fewer cases (4 for {single, streaming}-req × {single, streaming}-rsp instead of 9 for {empty, single, streaming}-req × {empty, single, streaming}-rsp).